lib/plugins/craft.js

```diff
@@ -8,184 +8,164 @@ function inject (bot, { version }) {
   const Item = require('prismarine-item')(version)
   const Recipe = require('prismarine-recipe')(version).Recipe
 
+  const craftingTableSlots = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+  const inventoryCraftingSlots = [1, 2, 3, 4]
+  const resultSlot = 0
+
   async function craft (recipe, count, craftingTable) {
```

Comment:
API.md says we accept a callback here, and clearly that isn't true and wasn't true before your commit. Can you update API.md to reflect that?

Reply:
Actually, we do: ``bot.craft = callbackify(craft)``
But if it was my decision I'd just yeet callbacks altogether ^^

Reply:
Oh derp, of course, we went over this already. Disregard my continued ignorance about how the hell this codebase works